web/content/js/versions.js

```diff
@@ -315,6 +315,94 @@ const navVersions = [
       }
     ]
   },
+  {
+    "title": "Foreman 3.8 - Katello 4.10 (supported)",
```

Comment:
We already have 3.8 as an RC (line 203). It should be sufficient to modify that line from RC to supported.

Reply:
whoops didn't realize that when I cherry-picked